Libs/GObject-2.0/Native/Classes/ObjectMapper.ToggleRef.cs

```diff
@@ -80,17 +83,31 @@ private void ToggleReference(IntPtr data, IntPtr @object, bool isLastRef)
                         _reference = weakObj;
                     else
                         throw new Exception("Could not toggle reference to strong. It is garbage collected.");
-
                 }
                 else if (isLastRef && _reference is not WeakReference)
                 {
                     _reference = new WeakReference(_reference);
                 }
+                
+                Debug.WriteLine($"Toggled ToggleRef: {GetLogState()}.");
```

Comment:
`GetLogState ` is executed in release mode. If we got this right one time it is obsolete.